script/convert_tags_to_topic.rb

```diff
@@ -54,4 +65,5 @@ def put_things_in_tag_to_correct_position cards, skip_year
   puts "#{source_cards_with_tag.size} note+tag cards found. Start Praying".green
   put_things_in_tag_to_correct_position source_cards_with_tag, true
   puts 'Finished source+tag updates!'.green
+  Card::Mailer.perform_deliveries = true
```

Comment:
Is the `Card::Mailer.perform_deliveries` a global setting? I though I need to re-enable it after the script is done.